src/main/java/io/wisetime/connector/jira/JiraConnector.java

```diff
@@ -208,6 +208,7 @@ public PostResult postTime(final Request request, final TimeGroup timeGroup) {
               )
       );
     } catch (RuntimeException e) {
+      log.error(e.getMessage(), e);
```

Comment:
Find it useful to log error so we could know what causing posting error by just reading the logs